functests/4_latency/latency.go

```diff
@@ -255,6 +228,65 @@ var _ = Describe("[performance] Latency Test", func() {
 	})
 })
 
+//read LATENCY_TEST_RUN environment var and parse it if set
+func getLatencyTestRun() {
+	latencyTestRunEnv := os.Getenv("LATENCY_TEST_RUN")
+	if latencyTestRunEnv != "" {
+		if latencyTestRunEnv == "true" {
+			latencyTestRun = true
```

Comment:
Thsi is still caching. The getter should just parse the value and return true/false. The `latencyTestRun` variable should be removed

Reply:
fixed, thanks.